src/components/connect.tsx

```diff
@@ -701,7 +701,7 @@ function connect<
         notifyNestedSubs,
       ])
 
-      let actualChildProps: unknown
+      let actualChildProps: uSES
```

Comment:
:hand: This code is internal and the type here won't matter much, but this _should_ be unknown.  Note that we're _calling_ `useSyncExternalStore` here and saving the result into `actualChildProps` - it's not a pointer to the hook itself.

Reply:
I rebased on master and pulled master. Not sure why it's showing as changing in my PR. It looks like that change was made here  https://github.com/reduxjs/react-redux/commit/fb8cfc0d76559d9ed028f26413e8845014ab51ee

Reply:
huh. @timdorr WHAT R U DOING?!?!?!? :)

Reply:
This is to fix typescript@next. You can't spread an unknown (which is done below this for the props on the component), which is no longer allowed. The value can't ever be anything other than a `uSES` because the only other option is for the useSyncExternalState to throw an error which is caught and then re-thrown. It either works and is a `uSES` or bails out of the function entirely.

Reply:
In that case, I think we ought to go with something like `Record<string, unknown>`.

The `uSES` type represents the actual `useSyncExternalStore` function, so it isn't the right type for the `actualChildProps` value.

Reply:
I agree that `Record<string, unknown>` is a more accurate type. I've made update in this PR

Reply:
Ah, gotcha. That is more correct.